Util/trapezoidal_speed_profile.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from RULEngine.robot import Robot, MAX_LINEAR_ACCELERATION, MAX_LINEAR_SPEED
+from Util import Position, Pose
+
+MIN_DISTANCE_TO_REACH_TARGET_SPEED = 0
+
+
+def get_next_velocity(robot: Robot, dt):
+    """Return the next velocity according to a constant acceleration model of a point mass.
+       It try to produce a trapezoidal velocity path with the required cruising and target speed.
+       The target speed is the speed that the robot need to reach at the target point."""
+    next_speed = robot.velocity.position.norm
```

Comment:
Nitpick: Should be rename `current_speed` and have different variable named `next_speed` which is initialized with `current_speed` value.

Reply:
can we redo that in another PR ?